thirdeye/thirdeye-frontend/app/pods/components/rootcause-dimensions-algorithm/component.js

```diff
@@ -239,35 +265,65 @@ export default Component.extend({
    * @private
    */
   _fetchIfNewContext() {
-    const { previousUrns, previousSettings } = this.getProperties('previousUrns', 'previousSettings');
-    const { mode, range, entities, metricUrn } = this.getProperties('mode', 'range', 'entities', 'metricUrn');
-    const newUrns = [...get(this, 'selectedUrns')];
+    const { mode, range, entities, metricUrn, previousUrns } = this.getProperties(
```

Comment:
Let's be consistent everywhere for `get`, `set` and `get/setProperties` everywhere in the file. I'm guilty of this too, been cleaning my act.

```
import {
  get,
  set,
  computed,
  setProperties
} from '@ember/object';```

Reply:
Sounds good!